miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -163,4 +163,13 @@ internal class MiniAppWebChromeClient(
         if (customView != null)
             onHideCustomView()
     }
+
+    override fun onShowFileChooser(
+        webView: WebView?,
+        filePathCallback: ValueCallback<Array<Uri>>?,
+        fileChooserParams: FileChooserParams?
+    ): Boolean {
+        miniAppFilePicker?.onShowFileChooser(filePathCallback, fileChooserParams, context)
+        return true
```

Comment:
Shouldn't this return the boolean from `MiniAppFilePicker`? 
```suggestion
        return miniAppFilePicker?.onShowFileChooser(filePathCallback, fileChooserParams, context) ?: false
```